tests/Application/config/routes.yaml

```diff
@@ -1,9 +1,3 @@
-bitbag_sylius_product_attribute_groups_plugin_shop:
-    resource: "@BitBagSyliusProductAttributeGroupsPlugin/Resources/config/shop_routing.yml"
-    prefix: /{_locale}
-    requirements:
-        _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
-
 bitbag_sylius_product_attribute_groups_plugin_admin:
     resource: "@BitBagSyliusProductAttributeGroupsPlugin/Resources/config/admin_routing.yml"
     prefix: /admin
```

Comment:
You can auto-solve this comment when you read it :P. Ff you have a case like a previous one (shop_routing and admin_routing) it's a good idea to create a file like `routes.yaml` in the `src/Resources/config` directory and wrap those two resource configuration inside it (with prefixing them). Then in application's `routes.yaml` you just import a single resource without worrying about prefixing it.